CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
   </summary>
 
 ### Minor
+  - Icon: Add new lightning icon
```

Comment:
Can you add this after the button changes and attach the PR number like the rest of these please